src/sentry/integrations/jira_server/webhooks.py

```diff
@@ -21,22 +21,29 @@ def get_integration_from_token(token):
     as Jira doesn't have any additional fields we can embed information in.
     """
     if not token:
-        raise ValueError("Token was empty")
+        return
 
     try:
         unvalidated = jwt.peek_claims(token)
     except jwt.DecodeError:
-        raise ValueError("Could not decode JWT token")
+        logger.info("jwt-decode-error")
```

Comment:
I'm not even sure if logging this is providing any value since there's no customer information to attach to it.